input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,506 @@
+AU Core is designed to ensure that core digital health and administrative data can be recorded, updated, searched, and retrieved efficiently, promoting interoperability and adoption across healthcare systems. By aligning with key FHIR implementation guides such [International Patient Access (IPA) 1.0.0)](https://hl7.org/fhir/uv/ipa/STU1/), [International Patient Summary Implementation Guide (IPS) 1.1.0](https://hl7.org/fhir/uv/ips/STU1.1/) and [US Core Implementation Guide 7.0.0](https://hl7.org/fhir/us/core/2024Jan/), AU Core ensures consistency with both international and national requirements.
 
-A resource may be conformant to multiple implementation guides. In addition to AU Core profiles, a FHIR resource may be conformant to profiles published in project implementation guides, other national implementation guides, or international implementation guides.
+Aligning data standards is crucial for enabling seamless data exchange across different healthcare systems and jurisdictions. It ensures that AU Core compliant systems can integrate smoothly with global healthcare frameworks, allowing consistent and accurate data sharing.
 
-The table below provides a high-level comparison, indicating if a resource conformant to a particular AU Core profile also satisfies (i.e. is conformant to) a profile from [International Patient Access](https://hl7.org/fhir/uv/ipa/STU1/), [International Patient Summary](https://hl7.org/fhir/uv/ips/STU1.1/), or [US Core](https://hl7.org/fhir/us/core/STU6.1/). See [Relationship to AUCDI and other IGs](https://build.fhir.org/ig/hl7au/au-fhir-core/branches/ft_blockvote-1/relationship.html#relationship-to-aucdi-and-other-igsl) for more information on the relationship of AU Core to these implementation guides.
+The comparison highlights where AU Core aligns with these national and international guides and identifies additional requirements necessary to meet their standards. 
 
-Conformance in reverse is not guaranteed, i.e. a resource conforming to [International Patient Access](https://hl7.org/fhir/uv/ipa/STU1/), [International Patient Summary](https://hl7.org/fhir/uv/ips/STU1.1/), or [US Core](https://hl7.org/fhir/us/core/STU6.1/) **MAY NOT** conform to AU Core.
+For more information on how AU Core relates to other implementation guides, see [Relationship with other IGs](relationship.html).
 
-<table border="1" cellspacing="0" cellpadding="0">
+### Profile comparison
+The profile comparison highlights how AU Core profiles align with other key FHIR implementation guides, such as International Patient Access (IPA), International Patient Summary (IPS), and US Core.  
```

Comment:
Replace the text between `### Profile comparison` and `**Legend:**` with:

```
As part of profile comparison, the requirements, constraints, and standards specified in a particular FHIR profile are evaluated. These requirements can include mandatory elements, cardinality constraints, data types, terminology bindings, usage rules, extensions, rules on missing or suppressed data. 

The table below provides a profile only comparison from AU Core to profiles in key implementation guides. Compliance in the reverse direction is not guaranteed, i.e. a resource that is compliant with an International Patient Access profile **MAY NOT** be compliant with AU Core.
```

Reply:
Done